drivers/include/periph/i2c.h

```diff
@@ -240,13 +240,13 @@ int i2c_release(i2c_t dev);
  * @param[out] data         memory location to store received data
  * @param[in]  flags        optional flags (see @ref i2c_flags_t)
  *
- * @return                  0 Success
- * @return                  -EIO Slave device doesn't ACK the byte
- * @return                  -ENXIO No devices respond on the address sent on the bus
- * @return                  -ETIMEDOUT  Timeout occurs before device's response
- * @return                  -EINVAL Invalid argument given
- * @return                  -EOPNOTSUPP MCU driver doesn't support the flag operation
- * @return                  -EAGAIN  Lost bus arbitration
+ * @return                  0 When success
+ * @return                  -EIO When slave device doesn't ACK the byte
+ * @return                  -ENXIO When no devices respond on the address sent on the bus
+ * @return                  -ETIMEDOUT  When timeout occurs before device's response
```

Comment:
There's an extra space before `When` here and after -EAGAIN also. Squash directly